xlators/mgmt/glusterd/src/glusterd-op-sm.c

```diff
@@ -2830,6 +2830,41 @@ glusterd_set_shared_storage(dict_t *dict, char *key, char *value,
     return ret;
 }
 
+static void
+glusterd_restore_volinfo_dict(glusterd_volinfo_t *volinfo, dict_t *orig,
+                              dict_t *dict)
+{
+    char *key = NULL;
+    int keylen;
+    int count;
+    int ret;
+    char keystr[50] = {
+        0,
+    };
+    char *value_orig = NULL;
+    char *value_current = NULL;
+
+    for (count = 1; ret != -1; count++) {
```

Comment:
`ret` is not initialized before the loop.

Reply:
Will fix.